test/api/repositories/template-detail/repository/.keboola/repository.json

```diff
@@ -16,6 +16,9 @@
           "version": "1.2.3",
           "description": "",
           "stable": false,
+          "components": [
+            "keboola.ex-aws-s3"
+          ],
```

Comment:
Akorát teda když odtud odstraním ty `components`, tak to v dopovědi nevrátí nic. Což asi není dobře?
![CleanShot 2022-05-31 at 12 51 19@2x](https://user-images.githubusercontent.com/215660/171157444-58de940a-4065-42b4-8ab0-122402a9d0c6.jpg)

Reply:
Ale nějak mi to nejde fixnout. 😅

Reply:
:eyes: